frontend/interfaces/config.ts

```diff
@@ -90,6 +90,8 @@ export interface IConfigFormData {
   enableSmtp: boolean;
   enableHostExpiry: boolean;
   hostExpiryWindow: number;
+  deleteActivities: boolean;
+  activityExpiryWindow: boolean;
```

Comment:
This interface could use some cleanup – it looks like it's just being used in the FleetDesktop settings card, and there is just having one field picked from it.